participants/Fouesneau/pyproject.toml

```diff
@@ -0,0 +1,74 @@
+# Starting with PEP 621 (June 2020), the Python community selected pyproject.toml as a
+# standard way of specifying project metadata. Setuptools has adopted this
+# standard and will use the information contained in this file as an input in
+# the build process.
+# https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+
+[build-system]
+requires = ["setuptools",
+            "setuptools-scm",
+            "wheel",
+            "extension-helpers"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ahw2022-pr-mf"
+description = "Exercise from AWH2022"
+version = "0.1.0"
```

Comment:
Should we strip the `setup.py` file from its content since it is all replicated here?